src/modules/numbers/entities/number.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({name: "UNIQUE_NUMBER_DEAL_FACILITY", database: "NUMBER_GENERATION", schema: "dbo", synchronize: false})
```

Comment:
do we need the `synchronize` property? I believe it's set up already at database level

Reply:
Removed, was here from POC.